tests/test_transforms.py

```diff
@@ -144,6 +144,18 @@ def test_wc_observations_chained(netinfodb, measurements):
 
     assert len(list(filter(lambda o: o.ip == "172.67.16.69", web_obs))) == 1
     assert len(web_obs) == 4
+    
+    msmt = load_measurement(
+        msmt_path=measurements[
+            "20221003005943.713064_IR_webconnectivity_ddfdfa11c9cf323d"
```

Comment:
I suggest sampling a few more measurements of wc 0.5 and adding checks that ensure ALL network events are being properly parsed (as opposed to just one of them)